faculty_cli/cli.py

```diff
@@ -1239,3 +1240,15 @@ def ls(project, path):
             click.echo("/project{}/".format(item.path))
         else:
             click.echo("/project{}".format(item.path))
+
+
+@cli.command()
+@click.argument("shell", type=click.Choice(["bash", "zsh", "fish"]))
+def completion(shell):
+    """Generate auto-completion scripts for faculty_cli."""
+    module_path = os.path.dirname(__file__)
+    click.echo(
+        open(
```

Comment:
I'd encourage using `open()` with a context manager instead of relying on the file descriptor getting closed implictly by the garbage collector.